benchmarks/kernels/benchmark_mixtral_moe.py

```diff
@@ -4,7 +4,7 @@
 
 os.environ['CUDA_VISIBLE_DEVICES'] = '0'
 
-from vllm.model_executor.layers.fused_moe import fused_moe
+from vllm.model_executor.layers.fused_moe.fused_moe import fused_moe, get_config_file_name
```

Comment:
Instead of doing this, you should expose `get_config_file_name` in `vllm/model_executor/layers/fused_moe/__init__.py` in the same way that `fused_moe` is currently exposed, and then do
```
from vllm.model_executor.layers.fused_moe import fused_moe, get_config_file_name
```

Reply:
Well, I did this because `get_config_file_name` is kind of private, and I think nobody else would use it.

Reply:
I'm open to change anyway. This won't hurt.

Reply:
Fixed in [b4a60f5](https://github.com/vllm-project/vllm/pull/3389/commits/b4a60f588ca0b07d9fa6cee9023e8f035c55bc76).